### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by jptaylor**

### CODE REVIEW

**Review by philmillman — APPROVED** [review]
as long as @regexident approves, then LGTM!

**Review by ablwr — APPROVED** [review]

**Comment by ablwr** [comment]
Yep, what Phil said

File: prebuilt-ui/ios-webview/ios-webview/WebViewPrebuilt/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+        // Make sure the webview's `<audio>` elements are
+        // allowed to auto-play inline:
+        configuration.allowsInlineMediaPlayback = true
+        // … and without user interaction:
+        configuration.mediaTypesRequiringUserActionForPlayback = []
+        
+        configuration.defaultWebpagePreferences = webpagePreferences
+
+        // We don't intend to ever show the webview, so we give it a zero-size:
```
Comment (regexident) [inline prebuilt-ui/ios-webview/ios-webview/WebViewPrebuilt/ViewController.swift:47]: This comment is obsolete, as unlike party-line we _do_ intend to show the webview.

File: prebuilt-ui/ios-webview/ios-webview/WebViewPrebuilt/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+        
+        // We pretend to be iOS Safari:
+        webView.customUserAgent = """
+            Mozilla/5.0 (iPhone; CPU iPhone OS 14_6 like Mac OS X) \
+            AppleWebKit/605.1.15 (KHTML, like Gecko) \
+            Version/14.1 Mobile/15E148 Safari/604.1
+        """
+
+        // Since the webview is off-screen anyway these are
```
Comment (regexident) [inline prebuilt-ui/ios-webview/ios-webview/WebViewPrebuilt/ViewController.swift:61]: This comment is obsolete, as unlike party-line the webview is _not_ off-screen.

**Review by regexident — CHANGES_REQUESTED** [review]

**Review by regexident — COMMENTED** [review]
:+1:

**Pull request merged**